x/bank/types/send_authorization.go

```diff
@@ -21,26 +19,25 @@ func NewSendAuthorization(spendLimit sdk.Coins) *SendAuthorization {
 
 // MethodName implements Authorization.MethodName.
 func (authorization SendAuthorization) MethodName() string {
-	return "/cosmos.bank.v1beta1.Msg/Send"
+	return sdk.MsgTypeURL(&MsgSend{})
```

Comment:
let's move it to a private (module) var. No need to recreate it. We can update other places as well, where we have similar functions.

Reply:
I tried this, but it actually doesn't work. `proto.MessageName(&MsgSend{}) == ""` on the module level. I'm not 100% sure how to make sure proto registration is run before defining a module var in this file.

See for example this failing test: https://github.com/cosmos/cosmos-sdk/pull/9139/checks?check_run_id=2467765703#step:6:155

Reply:
you can import like this:

`import _ "gitrhub.com/cosmos-sdk/x/bank/types"`

this will cause the proto init function to be run.

Reply:
Good point, thanks!

Which do you think is clearer? Adding `import _`, or just recreating a `sdk.MsgTypeURL()` in each function? I somehow feel the latter is better for readability, and less hidden knowledge.

Reply:
Maybe `RegisterInterfaces` can set these values?